ajax/libs/jodit/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "jodit",
   "version": "2.5.62",
-  "filename": "jodit.min.js",
+  "filename": "build/jodit.min.js",
```

Comment:
Let's reverse to `jodit.min.js`.
Because we define the base path as `build`, and we get the file from this folder.
So we just use `jodit.min.js` here.
Thank you